ckanext/superset/blueprints/superset.py

```diff
@@ -168,3 +168,34 @@ def list_databases():
         'superset_url': superset_url,
     }
     return tk.render('superset/databases_list.html', extra_vars)
+
+
+@superset_bp.route('/datasets', methods=['GET'])
+@require_sysadmin_user
+def list_datasets():
+    """ List all datasets created from Superset charts """
+    cfg = get_config()
+    sc = SupersetCKAN(**cfg)
+    # Obtener los datasets de Superset
+    datasets = sc.get_datasets()
+
+    # Procesar los datos de los datasets
+    super_datasets = []
+    for d in datasets:
+        if d is not None and isinstance(d, dict):
+            super_datasets.append({
+                'table_name': d.get('table_name', 'Sin nombre'),
+                'description': d.get('description') or '- - -',
+                'database_name': d.get('database', {}).get('database_name', 'Sin organización'),
+                'superset_chart_id': d.get('id'),
+                'private': d.get('private', False),
+                'resources': d.get('ckan_dataset', {}).get('name', '-'),
+            })
+        else:
+            log.warning(f"Elemento no procesado en superset_datasets: {d}")
+    superset_url = tk.config.get('ckanext.superset.instance.url')
+    extra_vars = {
+        'datasets': super_datasets,
+        'superset_url': superset_url,
+    }
```

Comment:
```suggestion

    superset_url = tk.config.get('ckanext.superset.instance.url')
    extra_vars = {
        'datasets': datasets,
        'superset_url': superset_url,
    }
```

¿Porque necesitamos todas esas correcciones?
De no ser estrictamente necesario usar el contenido tal como viene
Si es necesario algùn texto debe ser ingles